go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/fjl/gencodec v0.0.0-20220412091415-8bb9e558978c
 	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5
 	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff
-	github.com/gballet/go-verkle v0.0.0-20230224151942-0236f337641c
+	github.com/gballet/go-verkle v0.0.0-20230306134419-0d414350453b
```

Comment:
Using latest version of https://github.com/gballet/go-verkle/pull/333
After that one is merged, I'll switch here to the latest `go-verkle@master`.